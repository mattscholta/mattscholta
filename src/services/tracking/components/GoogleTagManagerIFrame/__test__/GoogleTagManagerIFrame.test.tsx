import * as React from 'react';
import { render, RenderResult } from '@testing-library/react';

import {
  GoogleTagManagerIFrame,
  GoogleTagManagerIFrameProps
} from '../GoogleTagManagerIFrame';

describe('GoogleTagManagerIFrame', () => {
  let props: GoogleTagManagerIFrameProps;
  let wrapper: RenderResult;

  const createWrapper = () => <GoogleTagManagerIFrame {...props} />;

  beforeEach(() => {
    props = { id: 'test-id' };
    wrapper = render(createWrapper());
  });

  it('matches snapshot', () => {
    expect(wrapper.container.childNodes).toMatchSnapshot();
  });
});
