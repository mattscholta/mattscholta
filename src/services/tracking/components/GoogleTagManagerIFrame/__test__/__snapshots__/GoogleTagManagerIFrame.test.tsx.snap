// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleTagManagerIFrame matches snapshot 1`] = `
NodeList [
  <noscript />,
]
`;
